### ISSUE TITLE
Kepub support?

### ISSUE THREAD

**Issue by LMPombar** [yshalsager/ebook-converter-bot#252]
I love the bot, it is super useful. 

However, as a Kobo user, I was wondering if you are planning on adding support for Kepub conversion? 

Thanks for the hard work.

**Comment by yshalsager** [comment]
> I love the bot, it is super useful.

Hello, Thanks for your kind words @LMPombar, glad to hear it's useful.

> However, as a Kobo user, I was wondering if you are planning on adding support for Kepub conversion?

The conversion is handled by [Calibre](https://manual.calibre-ebook.com/generated/en/ebook-convert.html#epub-output-options). I can't see KEPUB there.

**Comment by LMPombar** [comment]
You are absolutely right, I forgot I had to install an external plugin to do the conversion on Calibre Desktop. 

I'll try to think of a solution for it (maybe some collaboration with https://github.com/pgaskin/kepubify ?) 
Thanks again for your hard work.

**Comment by yshalsager** [comment]
> Wow, that'd be awesome. The plugin is called **KePub Output** [jgoguen/calibre-kobo-driver](https://github.com/jgoguen/calibre-kobo-driver)

I assume we only need input and output ones, right?

**Comment by LMPombar** [comment]
Yes, I assume KePub Input is to be able to convert Kepub into anything else, while KePub Output allows to convert anything into KePub. That would cover all the possible cases.

**Comment by yshalsager** [comment]
> Yes, I assume KePub Input is to be able to convert Kepub into anything else, while KePub Output allows to convert anything into KePub. That would cover all the possible cases.

Unfortunately, the plugin doesn't support CLI usage, thus it's not possible to add.

https://www.mobileread.com/forums/showpost.php?p=3456778&postcount=192

**Comment by yshalsager** [comment]
I have found an alternative approach, using kepubify from cli directly https://pgaskin.net/kepubify/docs/.

**Comment by yshalsager** [comment]
Added in f303a9e05fad51493f6c9ded4176d60ff9adba7a
